parcels/tools/exampledata_utils.py

```diff
@@ -79,24 +96,32 @@
 }
 
 
-example_data_url = "http://oceanparcels.org/examples-data"
+def _create_pooch_registry() -> dict[str, None]:
+    """Collapses the mapping of dataset names to filenames into a pooch registry.
 
+    Hashes are set to None for all files.
+    """
+    registry = {}
+    for dataset, filenames in EXAMPLE_DATA_FILES.items():
+        for filename in filenames:
+            registry[f"{dataset}/{filename}"] = None
+    return registry
 
-def get_data_home(data_home=None):
-    """Return a path to the cache directory for example datasets.
 
-    This directory is used by :func:`load_dataset`.
+POOCH_REGISTRY = _create_pooch_registry()
 
-    If the ``data_home`` argument is not provided, it will use a directory
-    specified by the ``PARCELS_EXAMPLE_DATA`` environment variable (if it exists)
-    or otherwise default to an OS-appropriate user cache location.
-    """
+
+def _get_odie(data_home=None):
+    if data_home is None:
+        data_home = os.environ.get("PARCELS_EXAMPLE_DATA")
     if data_home is None:
```

Comment:
So this would never be reached?

Reply:
No, basically if a user doesn't define a place to cache the data (i.e., `data_home`) it:

- Checks the environment variable `PARCELS_EXAMPLE_DATA`
- If env variable doesn't exist (i.e, data_home is still None) then it uses pooch to get a cache folder

This is the same as previous behaviour, so this PR should have no user observable changes (except now `get_data_home` is no longer a function)